vnext/Microsoft.ReactNative/XamlHelper.idl

```diff
@@ -10,6 +10,8 @@ namespace Microsoft.ReactNative {
   runtimeclass XamlHelper {
     XamlHelper();
     static Windows.UI.Xaml.Media.Brush BrushFrom(JSValueArgWriter valueProvider);
+    static Windows.UI.Color ColorFrom(JSValueArgWriter valueProvider);
+    static Boolean IsValidColorValue(JSValueArgWriter valueProvider);
```

Comment:
>IsValidColorValue [](start = 19, length = 17)

This function may encourage inefficient use where developers would parse the value twice: first to check that value is valid, and then to get the actual value.
A better pattern would be to return an optional value from the ColorFrom and then check if it is good or not.
I guess it can be expressed as using '?":

static Windows.UI.Color? ColorFrom(JSValueArgWriter valueProvider);

Also, please add the [noexcept2] attribute to make code more efficient.

Reply:
Updated